### PR TITLE
Add an Events page

### PR DESCRIPTION
**Pull request by castrojo**
This embeds a gcal widget onto a new Events page.

- Added new page and linked to from the navs, font-awesome icon included!
- Updated cloud custodian logo/text on the top left to always go to index.html and not just #
- There are nicer/cooler widgets for calendars on the web but chose to just embed this one to avoid new dependencies.
- We need a better method to have the community submit events, but deferring to just having them email me for now. 
- Updated URLs to the preferred upstream properties

cc @marcoceppi to check this since my html is rusty.

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<html>
```
Comment (marcoceppi) [inline events.html:1]:
```suggestion
<!DOCTYPE html>
<html lang="en">
```

File: events.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <meta name="description " content="Cloud Custodian ">
+        <meta name="author " content="Capital One ">
+
+        <title>Cloud Custodian</title>
+
+        <!-- Bootstrap Core CSS -->
+        <link href="css/bootstrap.min.css " rel="stylesheet ">
+
+        <!-- Custom CSS -->
+        <link href="css/landing-page.css " rel="stylesheet ">
```
Comment (marcoceppi) [inline events.html:16]: Additional space can cause issues on older browers.


```suggestion
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport " content="width=device-width, initial-scale=1">
        <meta name="description " content="Cloud Custodian events calendar">
        <meta name="author " content="The Cloud Custodian Authors">

        <title>Cloud Custodian</title>

        <!-- Bootstrap Core CSS -->
        <link href="css/bootstrap.min.css" rel="stylesheet">

        <!-- Custom CSS -->
        <link href="css/landing-page.css" rel="stylesheet">
```

File: events.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                                    <i class="fa fa-calendar"></i>
+                                    Calendar
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </nav>
+            </header>
+
+            <main role="main" style="padding-top: 5em">
```
Comment (marcoceppi) [inline events.html:74]:
```suggestion
            <main style="padding-top: 5em">
```

main is already main, a role stating as such is redundant

File: events.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        
+                <section>
+                    <iframe
+                    src="https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FDetroit&src=Y19naDhzc2pia3JvaWVxcWFzN2trbWgzdG9yc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23616161&title=Cloud%20Custodian"
+                    style="border:solid 1px #777" width="800" height="600"
+                    frameborder="0" scrolling="no" display:block></iframe>
```
Comment (marcoceppi) [inline events.html:80]:
```suggestion
                    <iframe
                    src="https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FDetroit&src=Y19naDhzc2pia3JvaWVxcWFzN2trbWgzdG9yc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23616161&title=Cloud%20Custodian"
                    style="border:solid 1px #777;  display:block;" width="800" height="600"
                    scrolling="no"></iframe>
```

block is a style element, not an HTML element.
frameborder is obsolete, use css which is currently set to 1px. If you want no border, set `border: 0`

File: events.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                </section>
+            <section>
+                <strong>Contact jorge@stacklet.io if you'd like to add a Cloud Custodian
+                related event to this community calendar.</strong>
+            </section>
+
```
Comment (marcoceppi) [inline events.html:86]:
```suggestion
        </main>
```

**Review by marcoceppi — COMMENTED** [review]
left a bunch of little HTMLisms. They're not bad, just things to make sure the page renders better.

File: events.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+                    <iframe
+                    src="https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FDetroit&src=Y19naDhzc2pia3JvaWVxcWFzN2trbWgzdG9yc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23616161&title=Cloud%20Custodian"
+                    style="border:solid 1px #777;  display:block;" width="800" height="600"
+                    scrolling="no"></iframe>
+                </section>
+            <section>
+                <strong>Contact jorge@stacklet.io if you'd like to add a Cloud Custodian
+                related event to this community calendar.</strong>
+            </section>
+        </main>
```
Comment (marcoceppi) [inline events.html:86]: Major nit: but spacing <3

```suggestion
                <section>
                    <strong>Contact jorge@stacklet.io if you'd like to add a Cloud Custodian
                    related event to this community calendar.</strong>
                </section>
            </main>
```

**Review by marcoceppi — APPROVED** [review]

**Review by kapilt — APPROVED** [review]
lgtm, thanks

**Comment by kapilt** [comment]
umm.. can we fold the one liner in #6886 into this its semantically for the exact same purpose, re events discoverability.

**Comment by kapilt** [comment]
nit, so rather than folks bookmarking /events.html it would be nicer to move to this a folder /events/index.html so the external link just becomes /events/

**Review by kapilt — APPROVED** [review]
lgtm, thanks

**Pull request merged**